test/gpu/pack_int8_args.cpp

```diff
@@ -52,6 +53,25 @@ void run_passes(migraphx::module& m)
                           migraphx::dead_code_elimination{}});
 }
 
+const std::unordered_set<std::string>& get_rocblas_fp32_archs()
+{
+    static std::unordered_set<std::string> supported_archs{"gfx908", "gfx90a"};
+    return supported_archs;
+}
+
+bool get_compute_fp32_flag()
```

Comment:
We should move this function to rocblas.hpp header and reuse the function from there. It should take a `context` as a parameter.

Reply:
Done.